app/controllers/hq/orphans_controller.rb

```diff
@@ -3,4 +3,63 @@ class Hq::OrphansController < HqController
   def show
     @orphan = Orphan.find(params[:id])
```

Comment:
Yes, use `load_orphan` is better.
I usually don't favor `before_action` for logic that is closely related to our actions. Makes it hard to read.
We should keep consistency with the other controllers.